src/libutil/unix/signals-impl.hh

```diff
@@ -1,5 +1,14 @@
 #pragma once
-///@file
+/**
+ * @file
+ *
+ * Implementation of some inline definitions for Unix signals, and also
+ * some extra Unix-only interfaces.
+ *
+ * (The only reason everything about signals isn't Unix-only is some
+ * no-op definitions are provided on Windows to avoid excess CPP in
```

Comment:
Not present in this PR, that's coming later?

Reply:
Yes